### PR TITLE
First pass at single image upload to s3

### PR DESCRIPTION
**Pull request by microfauna**
This PR adds the ability to upload a single featured-image.

- [ ] create tickets for future work
  - [ ] Add back multiple images and do away with preSigned upload URLs so we can preprocess images on the server.
  - [ ] Add image drag-and-drop rather than just "click to upload"

### CODE REVIEW

File: src/views/owner/EditBusiness.vue [hunk 1]
```diff
@@ -301,6 +305,29 @@ export default {
...
+
+      let form = new FormData()
+      for (let key in formData) {
+        form.append(key, formData[key])
+      }
+
+      form.append('file', e.target.files[0])
+      form.append('content-type', e.target.files[0].type)
+      await axios.post(postURL, form)
+      this.business.preview_image = formData.key
```
Comment (arecvlohe) [inline src/views/owner/EditBusiness.vue:330]: probably should do some error handling here I think

**Review by arecvlohe — APPROVED** [review]
I think error handling would be good but you can do a follow up PR on it

**Comment by arecvlohe** [comment]
recording

Right clicking and opening in new tab open to google maps. Weird!


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1440 x 792 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-49--native-owned.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1440x792&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-49--native-owned.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b450f58944b30007b1ebff&commentID=60b535db2cd90d60f81dbcdb&resolution=resolved)

**Comment by microfauna** [comment]
I fix the Maps / Link bug in the other PR, I'll add error handling there too as I refactor toast notifications!

**Pull request merged**